### ISSUE TITLE
Setup:static-content:deploy less compilation error

### ISSUE THREAD

**Issue by anethum** [Smile-SA/elasticsuite#259]
Hi,

I get the similar failure as menioted in #69 

Release 2.2.5 

frontend/Infortis/base/de_DE/css/styles-m.less
variable @autocomplete-item__border is undefined in file /../css/frontend/Infortis/base/de_DE/Smile_ElasticsuiteCore/css/source/_module.less in _module.less on line 30, column 34
28|                 border-bottom: 0;
29|             }
30|             .lib-css(border-top, @autocomplete-item__border);
31|             cursor: default;
32|             margin: 0;
33|             padding: @indent__xs @indent__xl @indent__xs @indent__s;

**Comment by afoucret** [comment]
Hi @anethum,

You can find a quite complete response for your problem :  #222.

TL;DR : you are using a custom theme that was written on an older version of Magento. Some standard variables have been changed. You shoud update your theme or append the missing variables since they are now standard in Magento themes.
